lib/diaspora/mentionable.rb

```diff
@@ -15,6 +15,7 @@ def self.mention_attrs(mention_str)
     del_pos = mention.rindex(/;/)
 
     name   = mention[0..(del_pos-1)].strip
+    name = nil if name.empty?
```

Comment:
So, why is `nil` better than an empty string?

Also, nit, something is wrong with the indentation.